content-src/components/LinkMenu/LinkMenu.js

```diff
@@ -12,13 +12,15 @@ const LinkMenu = React.createClass({
     };
   },
   userEvent(event) {
-    const {page, source, index, dispatch} = this.props;
+    const {page, source, index, dispatch, site} = this.props;
     if (page && source) {
       let payload = {
         event,
         page: page,
         source: source,
-        action_position: index
+        action_position: index,
+        url: site.recommended ? site.url : null,
+        recommender_type: site.recommended ? site.recommender_type : null
```

Comment:
kate said we're pretty much guaranteed a site, but to define ```site: {}``` in default props that way we don't have to keep checking if we have it. so i will do that